src/main/resources/logback.xml

```diff
@@ -0,0 +1,43 @@
+<configuration>
+    <appender name="JSON-STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <target>System.out</target>
+        <encoder class="net.logstash.logback.encoder.LoggingEventCompositeJsonEncoder">
+            <providers>
+                <message/>
+                <arguments/>
+                <logstashMarkers/>
+                <!-- Some fields are mapped to camel-case for consistency -->
+                <timestamp>
+                    <!-- Map default "@timestamp" to "timestamp" -->
+                    <fieldName>timestamp</fieldName>
+                </timestamp>
+                <loggerName>
+                    <!-- Map default "logger_name" to "loggerName" -->
+                    <fieldName>loggerName</fieldName>
+                </loggerName>
+                <logLevel>
+                    <!-- Map default "logLevel" to "severity" for cloud platforms such as GCP Stackdriver -->
+                    <fieldName>severity</fieldName>
+                </logLevel>
+                <stackHash>
```

Comment:
Is it really useful to see a hash value?

Reply:
It's useful to filter out the same logs by this hash. Details could be checked in here: https://github.com/logstash/logstash-logback-encoder/blob/master/stack-hash.md